python/lsst/sconsUtils/state.py

```diff
@@ -98,8 +98,8 @@ def _initVariables():
         SCons.Script.BoolVariable('force', 'Set to force possibly dangerous behaviours', False),
         ('optfile', 'Specify a file to read default options from', None),
         ('prefix', 'Specify the install destination', None),
-        SCons.Script.EnumVariable('opt', 'Set the optimisation level', 0,
-                                  allowed_values=('0', '1', '2', '3')),
+        SCons.Script.EnumVariable('opt', 'Set the optimisation level', 3,
+                                  allowed_values=('g', '0', '1', '2', '3')),
```

Comment:
gcc also allows `s` (optimise for size) and `fast` (disregards strict standards compliance); is it worth including these as well?